src/rgw/rgw_sal_sfs.h

```diff
@@ -111,10 +114,22 @@ class SFStore : public StoreDriver {
   ceph::mutex buckets_map_lock = ceph::make_mutex("buckets_map_lock");
   std::map<std::string, sfs::BucketRef> buckets;
 
+  // Signal shutdown condition to service threads
+  bool shutdown;
+
+  std::thread filesystem_stats_updater;
+  std::condition_variable filesystem_stats_updater_cvar;
+  std::mutex filesystem_stats_updater_mutex;
```

Comment:
why not use `ceph::mutex` here? we sort of lose lockdep capabilities that way, afaiu.

Reply:
:+1: